de/dev_setup/config_initial.md

```diff
@@ -28,7 +28,7 @@ The equipment below is highly recommended:
 
 ## Vehicle Configuration
 
-Install the [QGroundControl Daily Build](https://docs.qgroundcontrol.com/master/en/releases/daily_builds.html) for a **desktop OS**.
+Install the [QGroundControl Daily Build](https://docs.qgroundcontrol.com/master/en/qgc-user-guide/releases/daily_builds.html) for a **desktop OS**.
```

Comment:
Thanks very much. Can you undo all of these changes that are not in the /en path - i.e. the ones in ko, de etc. The reason is that the translated versions round trip from English through crowdin. I could accept these, but then it might well create merge conflicts with crowdin.

Reply:
Okay, I have undone all of these changes that are not in the /en path.